Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
  * Allow qualified access to events from other contracts.
 
 Compiler Features:
+ * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode. Official binaries are affected.
```

Comment:
```suggestion
 * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode.
```

I'd myself skip that part - it doesn't really belong into the changelog which describes code changes and not means of distributing. But fine either way.

Reply:
I want to make it clear that emscripten and native binaries produce different bytecode. You wanted me to make it short so I resorted to this.

How about this instead?
```suggestion
 * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode (especially native binaries vs emscripten binaries).
```

Reply:
That version sounds fine to me.